src/cli_shared/cli/mod.rs

```diff
@@ -150,6 +150,9 @@ pub struct CliOpts {
     /// Check your command-line options and configuration file if one is used
     #[arg(long)]
     pub dry_run: bool,
+    /// Skip loading actors from the actors bundle.
+    #[arg(long)]
+    pub skip_load_actors: bool,
```

Comment:
In what situations would we need to skip loading of actors bundle?

Reply:
If we don't intend to do any migrations and downloading the bundle is wasteful, then skipping the bundle makes sense. Also, it facilitates testing.